src/components/app-content-switcher/AppContentSwitcher.tsx

```diff
@@ -38,12 +38,7 @@ const AppContentSwitcher: FC<SwitcherProps> = ({
   return (
     <Stack sx={{ ...defaultStyles.stack, ...styles } as SxProps}>
       {renderBlock(switchOptions.left, active)}
-      <Switch
-        checked={active}
-        data-testid='switch'
-        onChange={onChange}
-        sx={defaultStyles.switch}
-      />
+      <AppSwitch data-testid='switch' onChange={onChange} />
```

Comment:
Why did you remove the `checked` prop here?

Reply:
Because it was redundant.